test_openai_compatibility.py

```
@@ -0,0 +1,207 @@
+    except Exception as e:
+        print(f"❌ Text completion failed: {e}")
+        return False
+
+def test_tools_integration():
+    """Test tools integration (PraisonAI extension)."""
+    print("\nTesting PraisonAI Tools Integration...")
+    
+    import requests
+    
+    try:
+        response = requests.post(
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)